UnityProject/Assets/Scripts/Items/IDCard.cs

```diff
@@ -216,21 +217,21 @@ public void SyncIDCardType(IDCardType oldCardType, IDCardType cardType)
 	}
 
 	/// <summary>
-	/// Checks if this id card has the indicated access.
+	/// Checks if this id card has the indicated clearance.
 	/// </summary>
 	/// <param name="access"></param>
 	/// <returns></returns>
-	public bool HasAccess(Access access)
+	public bool HasAccess(Clearance access)
 	{
 		return accessSyncList.Contains((int) access);
 	}
 
 	/// <summary>
-	/// Checks if this id card has the indicated access from a list of accesses.
+	/// Checks if this id card has the indicated access from a list of clearances.
 	/// </summary>
 	/// <param name="access"></param>
 	/// <returns></returns>
-	public bool HasAccess(List<Access> access)
+	public bool HasAccess(List<Clearance> access)
```

Comment:
Clearance framework already has a way to check for having a particular clearance. That's why migrating wasn't so straightforward as just replacing the type. There are some underlying design decisions that are radically different from ``Access``

Reply:
yeah, hard to do it all at once. But this PR removes all other uses of Access a part from AccessRestrictions script.

It'll be easier then to make progress to remove that now

Reply:
alright, fair